File: docker/javascript/entrypoint.js
// docker/javascript/entrypoint.js

const fs = require("fs");

(async () => {
  const event = JSON.parse(fs.readFileSync("/tmp/input.json", "utf8"));
  const { handler } = require("/function/function.js");
  const output = await handler(event);
  console.log(JSON.stringify(output));
})();

File: docker/python/entrypoint.py
# docker/python/entrypoint.py

import json
import importlib.util

def load_function():
    spec = importlib.util.spec_from_file_location("function", "/function/function.py")
    mod = importlib.util.module_from_spec(spec)
    spec.loader.exec_module(mod)
    return mod.handler

if __name__ == "__main__":
    with open("/tmp/input.json") as f:
        event = json.load(f)
    handler = load_function()
    output = handler(event)
    print(json.dumps(output))
